server/homebrew.api.js

```diff
@@ -57,22 +57,23 @@ const api = {
 						googleError = err;
 					});
 				// Throw any error caught while attempting to retrieve Google brew.
-				if(googleError) throw googleError;
+				if(googleError) {
+					const reason = googleError.errors[0].reason;
+					throw { ...googleError, HBErrorCode: reason == 'notFound' ? '02' : '01' };
+				}
 				// Combine the Homebrewery stub with the google brew, or if the stub doesn't exist just use the google brew
 				stub = stub ? _.assign({ ...api.excludeStubProps(stub), stubbed: true }, api.excludeGoogleProps(googleBrew)) : googleBrew;
 			}
 			const authorsExist = stub?.authors?.length > 0;
 			const isAuthor = stub?.authors?.includes(req.account?.username);
 			const isInvited = stub?.invitedAuthors?.includes(req.account?.username);
 			if(accessType === 'edit' && (authorsExist && !(isAuthor || isInvited))) {
-				throw `The current logged in user does not have editor access to this brew.
-
-If you believe you should have access to this brew, ask the file owner to invite you as an author by opening the brew, viewing the Properties tab, and adding your username to the "invited authors" list. You can then try to access this document again.`;
+				throw { name: 'Access Error', message: 'User is not an Author', status: 401, HBErrorCode: '03', authors: stub.authors };
```

Comment:
Can we make two different errors?

- The user is logged in, but not in the current authors.
- The user is *not* logged in at all (so the error doesn't say "The current logged in user....")

along the lines of: 
```suggestion
				if(req.account) {
					throw { name: 'Access Error', message: 'User is not an Author', status: 401, HBErrorCode: '03', authors: stub.authors };
				}
				else {
					throw { name: 'Access Error', message: 'User is not logged in', status: 401, HBErrorCode: '04', authors: stub.authors };
				}
```

Reply:
Implemented in commit [e28b6e7](https://github.com/naturalcrit/homebrewery/pull/2892/commits/e28b6e7a1926e88041ab8b4175b2128878f9a43e)